static/css/style.css

```diff
@@ -72,3 +72,31 @@ footer {
     color: var(--s-grey);
     background-color: var(--p-grey);
 }
+
+/* TEXT */
+.p-orange {
+   color: var(--p-orange);  

```

Comment:
Make sure you keep the indentation level consistend (4 spaces). Please adjust for anything below as well

Reply:
I'll go fix that up now